pilot/pkg/features/pilot.go

```diff
@@ -447,6 +447,9 @@ var (
 
 	EnableEnvoyFilterMetrics = env.RegisterBoolVar("PILOT_ENVOY_FILTER_STATS", false,
 		"If true, Pilot will collect metrics for envoy filter operations.").Get()
+
+	SortEnvoyFilterByName = env.RegisterBoolVar("SORT_ENVOY_FILTERS_BY_NAME", false,
+		"If enabled, envoy filters will be sorted by name allowing the dependency ordering taken care by name.").Get()
```

Comment:
Nit: allowing patch ordering by name. This can be useful when you have one patch that “depends on another” for ordering. In that case it is possible to use rc.d style 001-patch1 and 002-patch2 to enforce  patch application ordering.